src/Native/inc/unix/sal.h

```diff
@@ -2861,7 +2861,9 @@ of each annotation, see the advanced annotations section.
 #define __success(expr)                      _Success_(expr)
 #define __nullterminated                     _Null_terminated_
 #define __nullnullterminated
+#if !defined(_ARM64_)
 #define __reserved                           _SAL1_Source_(__reserved, (), _Reserved_)
```

Comment:
It would be better to rename this to `__clr_reserved` - like how this problem was fixed in CoreCLR.